cmd/deploy/remote.go

```diff
@@ -223,22 +223,23 @@ func (rd *remoteDeployCommand) createDockerfile(tmpDir string, opts *Options) (s
 }
 
 func (rd *remoteDeployCommand) createDockerignoreIfNeeded(cwd, tmpDir string) error {
+	dockerignoreContent := []byte(``)
```

Comment:
Can we add a comment explaining why do we create it empty

Reply:
done b91e64af29e071cbc55246556b66f024435f2ea3